RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -72,6 +72,14 @@ extension DeviceFactorSourceClient: DependencyKey {
 			)
 		}
 
+		let isSeedPhraseNeededToRecoverAccounts: @Sendable () async throws -> Bool = {
+			let deviceFactorSources = try await factorSourcesClient.getFactorSources(type: DeviceFactorSource.self)
+			let entities = try await deviceFactorSources.asyncMap {
+				try await entitiesControlledByFactorSource($0, nil)
+			}
+			return entities.contains(where: { !$0.isMnemonicPresentInKeychain })
```

Comment:
I don't know if this is the most optimal way to check this.

Reply:
But that logic is related to a particular account, isn't it? I want to know if there is any account for which the user needs to enter the seed phrase, in order to show the warning like this [mockup](https://app.zeplin.io/project/639c7e7ab391bb15ac5d25fa/screen/6616fbde81ea923357799a43)

Reply:
I mostly meant to be sure it is the same logic but for many accounts,
but it is fine, I just recalled that `isMnemonicPresentInKeychain` uses the same code to check for mnemonic presence.
So to me, your check here is fine 👍.